lib/Conversion/XTenToLinalg.cpp

```diff
@@ -342,6 +352,7 @@ class XTenConv2dOpConversion : public ConversionPattern {
                 loc, biasInitTensor.getType(), ValueRange{input, weight},
                 biasInitTensor, stridesAttr, dilationAttr)
             .getResult(0);
+    propagateLayerName(op, conv2dVal.getDefiningOp());
```

Comment:
@akafik - this is the one that was missing for the test case mentioned in the related issue.

Reply:
Yes.Propagate Layer name was missing for conv2d. Thanks @ljfitz